### DISCUSSION TITLE
Basic Structures of Matter - Supergravitation Unified Theory

### DISCUSSION THREAD

**Discussion by poelzi** [a-cosmology-group/acg#59] [Gravitation Models]
Dear fellow Scientists,

Those without confirmation bias and scientific scrutiny know, that the physical model we are currently using can not be true. Since QM and GR are inherent incompatible and QM in general failed on a very broad band of phenomena, despite what many physicists claim. High temperature superconductors, factual quantum hall effect, Rydberg state, water or even more complex molecules just to name some.
It is impossible to calculate protein folding with QM because it misses all the important details. No micro relativity and no proper electron orbits, no geometry in the core - just useless.
In cosmology things do not look better.

[Stoyan Sarg](https://www.researchgate.net/profile/Stoyan-Sargoytchev) did in my opinion the only sensible way out of this mess, first check our assumptions for newer research and drop those that are not convincing and build a model from scratch.
(my opinion: I'm absolutely convinced that he is the ignored genius of physics of our times - there are many details in his books and some circuits he designed, that are so utterly mind-blowing that I know, only geniuses of highest order can do that. But people seem care more about spelling then logic or other non scientific arguments...)

He was able to show, that you only need 3 basic assumptions:
1. Empty euclidean space without properties
    This means, no EM waves, no gravity, no particles without a rest mass, etc
    We know this as large black clouds in the cosmos. This does not mean they are empty per se
2. 2 types of fundamental particles.
   These are just very, very small balls of size relation 2/3. Something like a neutrino is large in this model. FP can only be found inside a super massive black hole or very, very far out in a galactic void.
3. On force of attraction, the supergravity force
   Similar to gravity but r³.
   I found this force much more convincing, since we are in a r³ space, so the force should fill a volume not a square. All forces from the standard model are emergent forces in BSM-SG. 

Enough of fundamental particles as bulk matter (structure inside a super-massive black hole), start a crystallization process those end state is a active galaxy as we know them. Galaxies are cyclic with 8-13 billion years in the active phase. Short gamma ray burst is the collapse of a galaxy, long gamma ray burst is the transition of the last phase of crystallization to active phase.

The model is super complex, but in my opinion, quite easy to understand. Some of the reasons are, that time is derived, which means you can stop time for analytic purposes (but never in reality), which simplifies everything.

Since cosmology is a result of nuclear physics or the model in general, most phenomena have a different origin. In the BSM-SG model, the cosmologic red-shift is derived effect from differences of the vacuum formed. Our vacuum is well structured build from neutrino decay. Those CL nodes are them self made from a smaller structure, called the prisms. Those prisms are them-self quite complex in their internal buildup. They implement the fine structure constant for example.
Those prisms have a slightly different size, every time they are formed in one phase of the galactic cycle.

This causes boundary surfaces where galaxies meet, and you have a suboptimal energy transfer which causes a red-shift. The red-shift periodicity effect is the result of this., the Super-massive Black hole is the absolute reference frame for our galaxy and the red-shift is a derived function with discrete states.

- The vacuum as we know it, with it's complex behavior is *non homogeneous*.
  You can see those discreete boundry surfaces everywhere once you know how they look. The hard clouds in Antanea galaxies (colliding galaxies), Crab Nebula (supernova from Cephid Type II)  or in evil eye galaxy are mixed CL space effects.
- Not all matter is formed the same. There a fundamental slight differences.
  You can test this with matter from another galaxy or globular cluster in a particle accelerator - systematically test meteorites for example.
- Protons are not small
  There are multiple problems with they way the size is derived. Small atoms can't explain the periodic table behavior.

Cepheid Type II stars for example, is a star from the previous cycle inside host CL space.
Globular Clusters are star formations with own CL space from previous cycle.

In general, BSM-SG is the first model I have encountered, that implement a sane relativity that Einstein would have approved.

Just some examples:
https://www.nature.com/articles/s41467-023-36074-2

This was predicted exactly like that. That is our kaon crystal, responsible for our magnetic field. It grows when planets reach the mass/volume ratio from Mercury. Then they collapse and build a magnetic field.
It is build from 2 types of kaons, one which you will not find on the standard model. 

This structure is the most important structure to understand cosmological effects. 
If it breaks horizontally, you get a supernova. If it is only covered in iron, you get a black hole.
If one side is uncovered you get a one jetted pulsar. If this is a star from a globular cluster and flies into the host galaxy, it looks like pillars of creation.
It this star would fly into another globular cluster, a quasar forms inside the tunnel created.

You can start with the [abstract](http://www.helical-structures.org/bsm_abs_paper.pdf), but I highly suggest reading the [full book](https://www.amazon.com/-/en/gp/product/1412083877/). It opened my eyes to nature and now everything makes sense.

I read some of the papers he published on his researchgate first, but did not really understand it good enough to be convinced. After understanding chapter 12 of his book, I was convinced that this is the implementation of nature. Maybe not all equations or all aspects like that, but the general buildup is correct. Complex not complicated.

I actually have slightly different views on some topics, but in general, the model is super strict and usually only allows 2-3 explanations for any given effect.

According to Stoyan, I have understood the model very well, because I asked him questions that are very specific and those only arise with deep understand according to him.

AMA

**Comment by khuramonline** [comment]
Following is from abstract:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/8c16436c-7ad2-4dd5-948b-a2c23a997383)

Here, application of commonsense logic is claimed. I have a sort of **[criteria](https://github.com/orgs/a-cosmology-group/discussions/60)** to judge the reliability of commonsense logic.  

And, apparently it seems that BSM theory is not derived from observed facts but rather based on already held ideas of larger reality. 

For example, it is already known (without observational basis) that (i) space is filled by such and such structures, (ii) Prism like particles also exist without observational basis. And many more such types of claims.

 What should be the right way to propose any particle?

Well, starting point must be observation. For example John Dalton had started from observations that observed chemical reactions take place between various elements only in specific quantitative ratios. A particle i.e. atom was proposed to account for or explain such observations.

**Reply by HanDeBruijn** [reply]
@poelzi . The way your post has started is actually very good. Indeed, few people know, for example, that the theory of superconductivity by Richard Feynman with his Cooper pairs has been a complete  failure. And I've never seen a decent QM account of the water molecule with its strange 104.5 angle between the hydrogen atoms. But the rest of your post escapes me a great deal.

**Reply by HanDeBruijn** [reply]
@khuramonline : it's Dalton, not Dolton. And yes, we already have enough particles for the centuries to come :-)

**Reply by poelzi** [reply]
> Here, application of commonsense logic is claimed. I have a sort of [criteria](https://github.com/orgs/a-cosmology-group/discussions/60) to judge the reliability of commonsense logic.

What he want's to say, that the BSM model uses classical logic and not mathematical one, which is much stricter. For example, a negative surface size or transforming a system into higher dimensional space, calculate something, transform it back into 3d. Or for example that the existence of a state is in relation to i. You can use complex numbers in a certain way to describe something easier, but you can not couple the existence of a particle to i.
These are degrees of freedom that are possible in mathematical logic, but not classical one.
In this aspect, BSM is a very strict model. You can also not invent more particles, in that sense it is complete and would be falsified with any more findings in LHC.

> And, apparently it seems that BSM theory is not derived from observed facts but rather based on already held ideas of larger reality.

More the contrary, it uses well established energies and masses of particles, constants and derives the geometry of our galaxies in a completely new way. I have not seen any model like this. Most constants are derived in this model, like the fine structure constant.

> For example, it is already known (without observational basis) that (i) space is filled by such and such structures, (ii) Prism like particles also exist without observational basis. And many more such types of claims.

It successfully predicted the "Higgs-Boson", but for a very different reason. The substructure of Neutrinos, the Prisms, are very very small. They can only measured indirectly, especially when you have of different kind. You can destroy matter from another galaxy or globular cluster or asteroid and you will see the formation of bubbles with a very specific red-shift behavior.
That this is "inconvenient" to test, is not a flaw of the model, physics is not wishful thinking.

This paper is a abstract of a 600 pages book. I needed a year to understand the model in detail and I would also not understand it from the abstract even with the more specific papers combined. I can assure you, that the way he derives one effect after another in a logical and straight way, is mind blowing.

This is kind of the perspective:
http://www.helical-structures.org/anal_algorithm.gif
That this is not transportable in 10 pages, should be understandable.

It is not that BSM just adds more particles, it is more a much higher level of detail.
The electron for example is not one particle, it is a system of 3 substructures, the electron shell, the positron and the tau particle. They together build the electron system with it's complex behavior and when you understand the internal buildup, the different energies at the  2 electron spin states make finally sense. Also how the electron system can be coupled to photon emission/absorption etc.

Superconductivity is a property of the vacuum, not the electron system. The electron system just disintegrates into electron shell - positron pairs. Roughly spoken: the fractual quantum hall effects is, that the magnetic field controls how long the electron positron train gets - each flat stair in the diagram is one electron more in the train.

In fact, it is fascinating how he later confirms with astronomical effects the internal buildup of the nucleus through phenomena like Quasars.
There is also a answer for Asteroids that have a anomalous rotational behavior.

@HanDeBruijn there are also many other "magic angles" that become totally clear once you understand the periodic table in BSM.
The fact, that there are isotopes with the same number of neutrons but different energy and at the same time not implementing any geometry is just absurd believe.
The fact that QM has problem describing any complex molecule is, that they never got rid of the Bohr perspective of a small nucleus and electrons in some space surrounding that nucleus. You need so many assumptions and believes to get a stable system out of that perspective and yet, you end up with something where you can just invent your wave function as you think until you find something that fits and call it a day.

**Reply by khuramonline** [reply]
@poelzi 

Classical logic is same as simple logic. I also compared simple logic with complex mathematical thought. Even if it is complex mathematical thought, it will not be reliable if starting point is already held (claim of) knowledge of whole reality or larger parts of reality like existence of certain structures in space and fundamental particles with details about their structures. These things are 'pre-known' in this 'model'.

Following is from abstract:

> The suggested physical models are successfully applied for explanation of dozens of physical effects and phenomena in a range from elementary particles to cosmological events.

That means suggested "physical models" are just pre-supposed to exist. And the observations are 'explained' on the basis of those pre-suppositions.

In fact, any ghost with suitable properties can be pre-supposed and then observations can be explained on the basis of the existence of such ghosts having those assigned properties.

**Reply by poelzi** [reply]
@khuramonline I can assure you, that his claim, that you only need 3 basic assumptions, I have already written those, are enough to explain galaxies / the universe. Empty euclidean space without properties, 2 types of fundamental particles and one law of attraction.

If you deny basic mechanical behavior for some reason, because this is what it actually is, the most detailed mechanical system I have ever understood. Then I don't know what to say, then we are on different spheres of truth deduction. I use the solomonoff induction and this model is the simplest you could possible build.
It is in the class of strong deterministic, classical models. You can have a Maxwell daemon in theory, but never in practice - which is the nicest of all possible answers.

You have to start somewhere when you explain a cycle and it makes sense to start where we are, this means the active phase of our galaxy. Same as our neighbor galaxies we can see.
At least, I was able to understand the processes he describes in chapter 12 how he envisions how these large structures we call protons/electrons/... are created. It is a multi step process which takes some billion years to finish. I personally have same own thoughts in this area that are to specific to discuss here, but yes, I think that the basic process he envisions there is correct.

In fact, it is the first model that could explain to me logically, how nature is able to create order out of fundamental chaos. Crystallization from smaller particles to larger ones. Due the simple fact that there is a twisting in a very early stage of crystallization process (QB1) results in enough complexity in the forces that you can explain everything.
Big Bang on the other hand, is just - magic. Too many constants over fitted with absurd math. It has not predicted a single thing correctly.

Also, some of the predicted phenomena where already found, for example, BSM predicted something that is called filaments on the top and bottom of the galactic nucleus (super massive black hole).
This stuff is funny because it behaves totally different to any matter we know. It has the maximum Newtonian mass possible ~ 10^26 N/m² (You have to calculate the fundamental surface first, which is this seldom case, is the object surface).
They where super happy to find dark matter, but of course, this would never be enough and ohh, on the wrong place as well. 
Anyways, you could rearrange those fragments and you could build a perfect spear that is slightly larger then the super massive black hole.
This is the roundest object you can ever find ')
In the volume between those 2 spheres, all the atom you are made from, me, all atoms in our current galactic cycle are once in there as precursors of protons/neutrons.
The space condition inside those spheres are quite different to our vacuum and can only exist there.
You will not understand this from my personal reflections on this, took me quite some time thinking about this object and it's life cycle. Now, this is for me one of the most beautiful object in the universe, it created us in some sense.

If you want to know where the fundamental particles come from, no idea. I personally think the small ones where created from the large ones in an irreversible compression process in the early stages of the universe. That is some 100s of billions of years ago. I think each cluster of the galactic filaments (how I love those name clashes....) was one of those processes.
But this is my personal working hypothesis of mine :)
Stoyan was able that you just need 2 types of size relation 2/3 and I'm pretty happy with that answer.

**Reply by khuramonline** [reply]
Ok. I can digest first assumption. But why are there two other assumptions? What is the background reasoning?

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
> ... the theory of superconductivity by Richard Feynman with his Cooper pairs has been a complete failure. And I've never seen a decent QM account of the water molecule with its strange 104.5 angle between the hydrogen atoms.

Wrong.
1. Feynman attempted to understand superconductivity but did not publish on it (see below).
2. Even if the Bardeen–Cooper–Schrieffer (BCS) model of superconductivity is incorrect, this does not discredit QM.  In the same way, every time an engineer designs a bridge that collapses doesn't mean that *mechanics* is discredited (it's the engineer's error).
3. Here is an explanation of how to [calculate the angle of the water bond](https://chem.libretexts.org/Courses/Pacific_Union_College/Quantum_Chemistry/10%3A_Bonding_in_Polyatomic_Molecules/10.02%3A_Hybrid_Orbitals_in_Water). This gives 109.5 degree, differing from the measured value because nobody knows how to calculate accurately the ratio of ***s*** and ***p*** character orbitals in such a complex system.  This is not a failure of QM but a failure of our ability to compute complex systems. (What are the mathematicians doing about this?)

This is also a problem in cosmology: The failures of the Big Bang model is unlikely a failure of QM or GR, but a failure of the models build up from these theories.  But most *cosmology dissidents* don't understand that (even the engineers).  Because they attribute the failures of models to a failure of a fundamental theory, they invent all sorts of [fictitious theories that never become widely accepted](https://cosmology.info/essays/models_marmet.html).

### From [Phys. perspect. 2 (2000) 30–47](https://web.njit.edu/~tyson/supercon_papers/Feynman_Superconductivity_History.pdf)

> **Feynman**: Oh, during that period I spent an awful lot of time trying to understand superconductivity. I did an awful lot of calculations and developed a lot of methods, which I’ve seen gradually developed by other people for other problems. But I didn’t solve the original problem that I was trying to solve, which was, where does superconductivity come from?
 And so I never published anything. But I have done an enormous amount of work on it. There’s a big vacuum at that time, which is my attempt to solve the superconductivity problem – which I failed to do.
> **Q**: But you did publish an article on superconductivity and superfluidity.
> **Feynman**: That was probably the result of some meeting where somebody asked me to give a summary of the situation or something.
> 
> It was indeed a meeting, the International Congress on Theoretical Physics, held at the University of Washington in Seattle, September 17–21, 1956,

**Reply by HanDeBruijn** [reply]
@marmetl . As a reply to your three issues:
1. [A Seminar on Superconductivity](https://www.feynmanlectures.caltech.edu/III_21.html) in _The Feynman Lectures on Physics_. Doesn't that count as a publication?
2. Could Quantum Mechanics have __predicted__ for example the [Periodic table](https://en.wikipedia.org/wiki/Periodic_table) of elements, instead of "explaining" it afterwards? __That__ is the question! I think that QM is half a theory, half technology; it's more like engineering than a science. Not that I hate engineering (I have been in the business myself :-).
3. Same remark for the [calculation of the angle of the water bond](https://chem.libretexts.org/Courses/Pacific_Union_College/Quantum_Chemistry/10%3A_Bonding_in_Polyatomic_Molecules/10.02%3A_Hybrid_Orbitals_in_Water). Suppose that we didn't already know about it. Could QM have __predicted then, in advance__, that the angle is not simply 180 degrees?

As a student, I've spent some time at the Chemistry department of the Eindhoven university, for half a year as I remember. Very instructive! Aiming to become a theoretical physicist, I didn't realize that _they didn't calculate_ spectral lines of a material, but simply looked them up in a (very large) book! And then I learned how they "calculate" chemical bonds. Oh my! That wasn't like the calculations we had been teached in our __BSM__ (Bouw en Structuur van de Materie) course. (Of course not :-)

**Reply by RedshiftDrift** [reply]
1. Not Feynman's ideas, only lectures on ideas of others (read the text by Feynman),
2. QM predicted new elements that were not on the periodic table and discovered later,
3. From QM, $s$-orbitals give 180 degrees, and $p$-orbitals give 90 degrees.  Since $H_2O$ does not have pure $s$ or $p$ character (the electronegativity of $H$ is strong enough to mix the $p$ character of the $O$ valence electrons with $s$ wavefunctions), the angle is predicted **not** to be 180 degrees nor 90 degrees, before even measuring it.

Many spectroscopists use 'patterns' in the spectra of various atoms/molecules to gain an understanding of the properties of molecules/solids.  They are forced to do this because the equations are not solved yet. (What are the mathematicians doing about this?)

**Reply by poelzi** [reply]
@marmetl this is one of the great mistakes of humanity, that we think there is particle physics, chemisty, astro physics,.... - no, there is only physics.  If your model does not cross validate the internal structure of the nucleus by cosmological effects, the model is most likely just overfitted over time. You can see this by the huge amount of constants with no real basis. The fine-structure constant for example. Every constant you are not deriving is a believe. The easiest metric for comparing models that are not falsified, is by counting all constants, all assumptions and all axioms - not a good metric but at least some.
In my opinion, every model that requires the axiom of choice is garbage from a physical point of view - this axiom makes sense in a mathematic space but not in the real world.

It is also a logical fallifasy to argu, that because model x predicted something correctly, that this model is now confirmed. No, it only falsified all models that did not predict this case and there is a overcountable infinite set of theories that predicted exactly this behavior - we just don't know all of them.

As somebody who studied philosophy and physics, I'm always puzzeled by the fact, that people can become a scientists, without spending at least some semesters in philosophy of science....

**Comment by RedshiftDrift** [comment]
>  ... the physical model we are currently using can not be true. Since QM and GR are inherent incompatible and QM in general failed on a very broad band of phenomena

**This is incorrect.**  QM was not designed to address gravitation, and GR was not designed to address quantum effects.  It is incorrect to try to use these theories together (e.g. black hole quantum evaporation) because that's stepping outside the range of phenomena for which they are individually valid.

QM is successful on an impressive range of phenomena in atomic, molecular and nuclear physics, quantum optics, organic chemistry and biology.  Technological applications like MRI scanning, the laser, quantum cryptography and interpretation of most astronomical data are all examples of the success of QM.

GR is also successful on describing relativistic particle physics, the fine atomic structure and other relativistic effects from atomic dynamics to planetary motion to relativistic plasma physics and other astronomical phenomena.

The fact that QM and GR are incompatible does not mean they cannot be true, it just means they each solve different problems.

**Reply by HanDeBruijn** [reply]
@marmetl .
_GR is also successful on describing relativistic particle physics, the fine atomic structure and other relativistic effects from atomic dynamics_.

I think you've made a typo. Or you are mixing up Special Relativity (SR) with General Relativity (GR).
SR and GR are not at all similar. They are very different theories. GR is not useful at all in combination with atomic physics. While SR, as you (meant to) say correctly, leads to results significantly different from QM in its classical setting.

As a student, I've once derived the Dirac equation. Talking about _the fine atomic structure_, with the Dirac theory one calculates the relativistic energy levels for the hydrogen atom; for $Z=1$ in:

$$
E = m c^2 \left[ \left( 1 + \frac{ (Z.\alpha)^2 }
   { \left[ \sqrt{ (j+ \frac{1}{2} )^2 - (Z.\alpha)^2 } + n \right] ^2 }
   \right)^{-1/2}-1\right]
$$

Where: $m=$ electron (rest) mass, $c=$ speed of light, $Z=$ atomic number, $\alpha=$ fine-structure constant, $j+\frac{1}{2} \geq 1$ and $n,j=$ positive integer numbers.
Now the weird thing is that the relativistic formula _does not explain_ the energy levels for the hydrogen atom - as they are actually measured - in any way better than the classical formula. Can someone explain then where it is good for?

**Reply by ftessier** [reply]
There is no spin with the classical approach.

**Reply by ftessier** [reply]
About the Special Relativity (SR) and General relativity (GR) distinction: it might border on philosophy, but SR theory is strictly restricted to inertial frames of reference. However our observations of the world give us a rather convincing indication that it involves frames that are _not_ inertial. Hence, admitting SR essentially begs for a more general theory that admits SR as a local limit. That is a tall order, and to my knowledge (admittedly limited), GR has been the only consistent proposition so far. In that sense, I am not sure distinguishing SR and GR is substantial here.

**Reply by poelzi** [reply]
Neither SR nor GR are implemented with Einstein formulas - they should be seens as approximations only. For example: The effect of photons experiencing a red-shift when sent from the surface of a planet and measured outside the gravitational well, is explained by CL node distance changes. On the other extreme, the node distance reaches 0 on the extremity of a super massive black hole. Classical relativity breaks down because the underlaying relationship of node distance, CL-node resonance frequency and time is hidden in field equations, so there are huge differences there. Also, c is NOT the primary time etalon. The primary time does not stop near a super massive black hole, only the vacuum viscosity reaches 0 and therefore every process that requires a fluid vacuum will stop. This means every effect that depends on EM waves, electron system, neutron spin, ... so nearly everything except gravity propagation and some other effects.

Also, all spin phenomena are nicely explained due the complex buildup of electrons/neutrons/protons in a very classical way. The 2 energy states of the electon - it aways puzzled me, now it is totally clear why.
